test/redirects.js

```diff
@@ -49,6 +49,7 @@ const HTTP_TESTS = [
   ['http://validator.whatwg.org/', 301, 'https://validator.whatwg.org/', 'keep'],
   ['http://webvtt.spec.whatwg.org/', 301, 'https://webvtt.spec.whatwg.org/', 'keep'],
   ['http://whatwg.org/', 301, 'https://whatwg.org/', 'keep'],
+  ['http://whatwg.org/c', 301, 'https://whatwg.org/c', 'keep'],
```

Comment:
@annevk, looking at the output (tests pass) I see that we were already testing http://whatwg.org/foo due to the 'keep' bit. Did you know that, and is the additional test needed?

Reply:
I did not, though it might be good to test this in addition since /c is a redirect itself.

Reply:
Sure, let's keep it.